pages/weather/index.js

```diff
@@ -0,0 +1,108 @@
+import Layout from "../../components/Layout.js";
+import { useState } from "react";
+import styled from "styled-components";
+
+const api = {
+  key: "0a1b4079174262c9bd54e2c2800cda16",
```

Comment:
Are you sure that this key should be exposed to the user? If this is your private key (which is now exposed and should be replaced by a new one), please consider to use a `.env` file and take the secret value from there.